overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
```

Comment:
I think we can just define this as `override`, and subsequently each of the elements in the type lower case as well (e.g., `cidr`, `city`, `country`) since this identifier does not need to be exported.

Reply:
Makes sense, I've added that in https://github.com/ameihm0912/geomodel/pull/10/commits/74861a055de46f1c2b5aeabc2ae758030161f1b0